app/styles/libs/_helpers.scss

```diff
@@ -140,10 +140,15 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 }
 
 .social-link-button {
-  width: 50px;
-  height: 50px;
+  width: 48.66px;
+  height: 48.66px;
   max-width: 50px;
   max-height: 50px;
+  margin-right: 2px !important;
+
+  &:nth-child(5n) {
```

Comment:
why?

Reply:
Mam Actually if you see in the previous screenshots then the reason for not aligning the button box completely with others was a small space coming due to the margin of a button. So the problem was that there should be a margin other than that of a large button, So handling the above problem statement I made the margin of the last button equal to zero in order to remove that small space.  I think this clears your doubt mam, If not please let me know.

Reply:
I should work for all resolutions
![Screenshot from 2021-03-04 17-38-52](https://user-images.githubusercontent.com/61330148/109961897-90f73180-7d10-11eb-8fb9-5991e95cf103.png)

Reply:
Mam, The dynamic approach which I was following was leaving some unnecessary space at which Mario Sir suggested in the initial comments that it is not possible in every resolution but should work in at least one resolution, After which I tried 3 different ways to tackle this issue, the best one which I have got is this one and unable to think of any better approach, So Mam could you please help me with this, Like approach to make it equal for all resolutions. If possible please guide me, Mam.

Reply:
Yeah it works for me at 100% resolution